query-node/mappings/src/scheduler.ts

```diff
@@ -0,0 +1,56 @@
+import { In } from 'typeorm'
+import { DatabaseManager, StoreContext } from '@joystream/hydra-common'
+import { eventEmitter, ProcessorEvents } from '@joystream/hydra-processor/lib/start/processor-events'
```

Comment:
@Lezek123 because the runtime doesn't implement the events necessary to map the transition between funding, work, judgment, and failed stages, I have added this "scheduler" logic based on the hydra `eventEmitter`. According to you would this be an acceptable solution ?